### ISSUE TITLE
Usage with reason-react and React.forwardRef

### ISSUE THREAD

**Issue by WhyThat** [aantron/bisect_ppx#375]
Hello there,
I'm trying to setup bisect_ppx with on a project which use reason-react

Unfortunately when I try to build with `BISECT_ENABLE=true` I got this error `Fatal error: exception Invalid_argument("react.component calls can only be on function definitions or component wrappers (forwardRef, memo).")` this error is throw on component who are using `React.forwardRef` 

I've created a repo to reproduce the error:
https://github.com/WhyThat/bisect-with-forward-ref

Is there a workaround to have bisect_ppx working with component which are using `React.forwardRef`?

Cheers!

**Comment by aantron** [comment]
Hi @WhyThat, sorry for the big delay.

I was able to get the repo building with this change:

```diff
-[@react.component]
+[@react.component] [@coverage off]
 let make = React.forwardRef(ref => {React.null});
```

This uses the [coverage suppression attributes](https://github.com/aantron/bisect_ppx#controlling-coverage-with-coverage-off). It suppresses coverage for the whole AST under `make`, which, I assume, is not what you want &mdash; you probably want subexpressions instrumented, just not the top level. You can work around that immediately by factoring out the body of `forwardRef`.

How often does this occur in your real code? If this is common, I should probably add either a variant of `[@coverage off]` that applies only to the current expression and not subexpressions, or treat `React.forwardRef` as a special case.

**Comment by WhyThat** [comment]
Hey !
Indeed, adding `[@coverage off]` deals with the error but as you presume I would like to have subexpressions instrumented.
In fact as my team and I are building an UI library `React.forwardRef` is used almost on all components.
Adding a `next-line-off` variant could be enough IMO, but if you can treat `React.forwardRef` this could be awesome !

**Comment by aantron** [comment]
I've fixed it in `master` and tagged 2.6.1, but there is some upstream problem with the binaries build. I'll do a release as soon as I can get that fixed, and I'll ping this issue.

**Comment by aantron** [comment]
It should be installable from npm now.
